src/json/lang.ts

```diff
@@ -41,6 +44,7 @@ export default {
 		livescript: 'LiveScript',
 		lua: 'Lua',
 		markup: 'Markup',
+		md: 'Markdown',
```

Comment:
What for is it? It will result in double entries in the list and it won't work as there are no corresponding components in Prism. So you will have to map md value to markdown file.

<img width="1620" alt="Screenshot 2024-07-05 at 09 01 48" src="https://github.com/anyproto/anytype-ts/assets/1450818/e4056399-6119-465f-a324-e883084b5bd1">
https://github.com/PrismJS/prism/tree/master/components

Reply:
Yes, this would be much better and this way it would be possible to find the correct option by alias and not double options with the same name. You need to check if this code works for the alias in the text block:

```
let lang = fields.lang;
			let grammar = Prism.languages[lang];

			if (!grammar && (lang != 'plain')) {
				lang = J.Constant.default.codeLang;
				grammar = Prism.languages[lang];
			};
```

Reply:
Because I think this map `Prism.languages` is built upon the list of required lang files, to be precise. So it may not work and you need to manually map alias to the lang file in this logic.